telecarla_gui/source/gui/src/sdl_event_parser.cpp

```diff
@@ -60,7 +60,7 @@ void SDL_EventParser::pollEvents()
             }
             else if (event.key.keysym.sym == SDLK_q)
             {
-                carlaEgoVehicleControl_.gear = carlaEgoVehicleControl_.reverse ? 1 : -1;
+                carlaEgoVehicleControl_.gear = carlaEgoVehicleControl_.reverse != 0U ? 1 : -1;
```

Comment:
Reverse is already defined as bool https://github.com/carla-simulator/ros-carla-msgs/blob/master/msg/CarlaEgoVehicleControl.msg#L26

Reply:
Yeah, that's what I saw as well. Just wanted to show what clang-tidy thinks